ssl-config-core/src/main/resources/reference.conf

```diff
@@ -91,6 +91,9 @@ ssl-config {
 
       # The data for the keystore. Either this must be non null, or path must be non null.
       data = null
+
+      # The password for loading the truststore. If null, uses no password.
+      password = null
```

Comment:
Maybe add a comment that it is recommended to pass in the password via env variables instead of putting it here directly?